runestone/__init__.py

```diff
@@ -41,6 +41,7 @@ def runestone_static_dirs():
     module_static_css.append(os.path.join(basedir, "accessibility", "css"))
     module_static_css.append(os.path.join(basedir, "webgldemo", "css"))
     module_static_css.append(os.path.join(basedir, "matrixeq", "css"))
+    module_static_css.append(os.path.join(basedir, "lp", "css"))
```

Comment:
I need to think this through a bit...  whatever we add to static dirs means that sphinx will copy the files from the path to the static folder... mostly this is redundant as nearly all of the files in the css/js folders for a component are bundled and do not need to be copied.

Reply:
True -- but in this case, https://github.com/bjones1/CodeChat/blob/master/CodeChat/css/CodeChat.css#L31 performs the include. This file isn't processed by webpack, so it needs to be saved as a standalone file for inclusion.

Reply:
But is there some reason why we can’t include it in webpack?  The goal is to minimize the number of individual scripts and css links when they can be just as easily imported by webpack.

Reply:
I don't know. I see an error at https://github.com/bjones1/CodeChat/blob/master/CodeChat/css/CodeChat.css#L31 when that imports fails, so I'm assuming it doesn't work...

That is, can CSS not included in the webpacked file still do @imports as usual? I don't think so...

Reply:
I have css-loader configured, but I think I failed to enable it for css, just for png,jpg,svg...  https://webpack.js.org/loaders/css-loader/

Just add css to line 46 of webpack.config.js and see if that does it.